styles/google-gemini/catppuccin.user.css

```diff
@@ -232,9 +254,19 @@
 
     --og-theme-color: @text;
 
+    .new-feature {
+        background: @surface2;
+    }
+    
     .button-pressed {
       --bard-color-primary: @mantle;
     }
+    
+    .textarea, .text-input-field {
+        background-color: @mantle;
+        color: @text;
+    }
+    
```

Comment:
What the point of this? input field is already styled using --gem-sys-color--surface-container

Reply:
that's so when you click on the text box it doesn't change color

Reply:
Why, does it need to do that? in original gemini without catppuccin, its default behaviour to change colors to more lighter.

i see no reason for it to not do that in catppuccin.

Reply:
i thought it didn't do that

Reply:
anyways im going to change that